web/src/compositions/useVersion.ts

```diff
@@ -72,9 +72,9 @@ export function useVersion() {
       if (usesNext) {
         latest = versionInfo.next;
       } else if (current.includes('rc')) {
-        latest = versionInfo.rc;
```

Comment:
@6543 Or should we directly do this in the ci config file updating the version.json file.

Reply:
I want to release a v2.1.1 anyway ... :)

and later i suggest to use a version parse lib

Reply:
#3041